index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Should I bring a jacket?</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Fredoka+One&family=Montserrat&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" type="text/css" href="styles/index.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Should I bring a jacket 🧥?</h1>
+      <span>
+        displaying for the city of
+        <span id="cityNameHeader"></span>
+      </span>
```

Comment:
this works fine! something more common you might see is this:

```suggestion
      <h3>
        displaying for the city of
        <span id="cityNameHeader"></span>
      </h3>
```

nested `span` elements don't occur often. Usually you would pick a `p` or heading element, then nest the `span` inside!